src/components/Data/Collections/CreateOrUpdate.vue

```diff
@@ -14,7 +14,7 @@
       :steps="['Mapping', 'Form']"
       @changed-step="setEditionStep"
       class="card-panel card-header">
-    </stepper>
+    </stepper>²
```

Comment:
I didn't know we could use ² to multiply div in html :D
```suggestion
    </stepper>
```